cvat-core/src/server-proxy.js

```diff
@@ -249,12 +249,19 @@
                 }
             }
 
-            async function validateTask(id) {
+            async function validateTask(id, jobs) {
                 const { backendAPI } = config;
 
+                let query = "";
+                if (jobs !== null) {
+                    const queryParams = new URLSearchParams();
+                    queryParams.append("jobs", jobs);
```

Comment:
nit: it is useful to keep links human-readable
you can use `jobs.join()` here

Reply:
Do you mean not using `URLSearchParams` at all and formatting `jobs` right into url? That way:
``` `${backendAPI}/tasks/${id}/validate?jobs=${jobs.join(',')}` ```

Reply:
You can still use `URLSearchParams` if you wish. But the param value can be `jobs.join()` to achieve the same effect.